### DISCUSSION TITLE
Welcome to NSPanel Pro Tools Discussions!

### DISCUSSION THREAD

**Discussion by seaky** [seaky/nspanel_pro_tools_apk#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eerke2021** [comment]
if you want to push the update over adb to the roottool what is the command and does it need zipping first?

**Reply by eerke2021** [reply]
I wasted much time already guessing badly

**Reply by eerke2021** [reply]
it was already zipped. That was it. Now I just misremembered system as software in name.

**Reply by eerke2021** [reply]
all sorted. Just had to add .zip to the end and rename file from software_update to system_update
